src/Neo.VM/ReferenceCounter.cs

```diff
@@ -30,9 +33,20 @@ public sealed class ReferenceCounter
         /// </summary>
         public int Count => references_count;
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static bool NeedTrack(StackItem item)
+        {
+#pragma warning disable CS0162
+            if (TrackAllItems) return true;
```

Comment:
Does we need this private const?

Reply:
Keep it for now, the optimizer will take care of it.

Reply:
We can add a `#if DEGUG`

Reply:
It's not a debug thing.